### PR TITLE
Add rules page

### PR DESCRIPTION
**Pull request by jlholm**
Initial setup of the rules page. We can add onto it later with more concrete ruling if necessary. These are very basic and general rules. Thinking we should add an advanced section for more competitive players (myself)

### CODE REVIEW

File: web/templates/layout/app.html.eex [hunk 1]
```diff
@@ -22,6 +22,7 @@
         <section class="navbar-section">
           <%= link "Matches", to: "/matches", class: "btn btn-link" %>
           <%= link "Players", to: "/users", class: "btn btn-link" %>
+          <%= link "Rules", to: "/rules", class: "btn btn-link" %>
```
Comment (Serneum) [inline web/templates/layout/app.html.eex]: Instead of hardcoding the URL, use `page_path(@conn, :rules)`

File: web/templates/page/rules.html.eex [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  <h3>Legal Serve</h3>
+  <p>
+    The ball must rest on an open hand palm. Then it must be tossed up at least 6 inches and struck so the ball first bounces on the server's side and then the opponent's side.
+  </p>
+  <p>
+    If the serve is legal except that it touches the net, it is called a let serve. Let serves are not scored and are reserved.
+  </p>
+  <h3>Alternate Serving</h3>
+  <p>
+    After each 5 points have been scored the receiving player or pair shall become the serving player or pair and so on until the end of the game, unless both players or pairs score 20 points or the expedite system is in operation, when the sequences of serving and receiving shall be the same but each player shall serve for only 1 point in turn.
```
Comment (Serneum) [inline web/templates/page/rules.html.eex]: These sound super legal/formal. Should we make the text of the rules more user-friendly?

Comment (jlholm) [reply web/templates/page/rules.html.eex]: Lol, I just copy/pasted that from the site you linked. I'll change it

Comment (Serneum) [reply web/templates/page/rules.html.eex]: That was my assumption. Some of them are definitely hard to read

**Review by Serneum — APPROVED** [review]
LGTM

**Pull request merged**